tests/actor/test_battery_status.py

```diff
@@ -597,6 +599,68 @@ async def test_sync_blocking_interrupted_with_invalid_message(
 
         await tracker.stop()
 
+    @time_machine.travel("2022-01-01 00:00 UTC", tick=False)
+    async def test_timers(
+        self, mock_microgrid: MockMicrogridClient, mocker: MockerFixture
+    ) -> None:
+        """Test if messages changes battery status/
+
+        Tests uses FakeSelect to test status in sync way.
+        Otherwise we would have lots of async calls and waiting.
+
+        Args:
+            mock_microgrid: mock_microgrid fixture
```

Comment:
mocker is missing if you add `Args` in tests, probably not needed...

Reply:
Done